### PR TITLE
tests: build_all: sensor: enable iis328dq and wsen-pads

### PR DESCRIPTION
**Pull request by soburi**
wsen-pads:

Fix to eliminate warnings caused by type mismatch.

iis328dq:
Separating declarations of variables not referenced with #ifdef
when CONFIG_IIS328DQ_THRESHOLD is disabled

### CODE REVIEW

File: drivers/sensor/st/iis328dq/iis328dq_trigger.c [hunk 1]
```diff
@@ -175,8 +175,10 @@ static void iis328dq_handle_interrupt(const struct device *dev)
 	struct iis328dq_data *data = dev->data;
 	stmdev_ctx_t *ctx = (stmdev_ctx_t *)&cfg->ctx;
 	iis328dq_status_reg_t status;
+#ifdef CONFIG_IIS328DQ_THRESHOLD
```
Comment (jeppenodgaard) [inline drivers/sensor/st/iis328dq/iis328dq_trigger.c:178]: The variables should probably be marked as `__maybe_unused`. See https://www.kernel.org/doc/html/latest/process/coding-style.html section 21.

Comment (soburi) [reply drivers/sensor/st/iis328dq/iis328dq_trigger.c]: Thank you for your review.
I applied your suggestion.

**Review by jeppenodgaard — DISMISSED** [review]

**Review by avisconti — DISMISSED** [review]
LGTM

File: drivers/sensor/st/iis328dq/iis328dq_trigger.c (hunk 1 above)
Comment (MaureenHelm) [reply drivers/sensor/st/iis328dq/iis328dq_trigger.c]: Zephyr convention is to use the ifdef

File: drivers/sensor/st/iis328dq/iis328dq_trigger.c [hunk 2]
```diff
@@ -175,8 +175,8 @@ static void iis328dq_handle_interrupt(const struct device *dev)
 	struct iis328dq_data *data = dev->data;
 	stmdev_ctx_t *ctx = (stmdev_ctx_t *)&cfg->ctx;
 	iis328dq_status_reg_t status;
-	iis328dq_int1_src_t sources1;
-	iis328dq_int2_src_t sources2;
+	__maybe_unused iis328dq_int1_src_t sources1;
+	__maybe_unused iis328dq_int2_src_t sources2;
```
Comment (MaureenHelm) [inline drivers/sensor/st/iis328dq/iis328dq_trigger.c:179]: Zephyr convention is to use an ifdef

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

File: drivers/sensor/st/iis328dq/iis328dq_trigger.c (hunk 2 above)
Comment (soburi) [reply drivers/sensor/st/iis328dq/iis328dq_trigger.c]: Restored.
Thank you for your check.

**Review by MaureenHelm — APPROVED** [review]

**Review by ubieda — APPROVED** [review]

**Pull request merged**